go/client/cmd_device_remove.go

```diff
@@ -49,14 +53,35 @@ func (c *CmdDeviceRemove) Run() (err error) {
 
 	return cli.RevokeDevice(context.TODO(), keybase1.RevokeDeviceArg{
 		Force:    c.force,
-		DeviceID: c.id,
+		DeviceID: id,
 	})
 }
 
+func (c *CmdDeviceRemove) lookup(name string) (keybase1.DeviceID, error) {
+	cli, err := GetDeviceClient()
+	if err != nil {
+		return "", err
+	}
+	if err := RegisterProtocols(nil); err != nil {
+		return "", err
+	}
```

Comment:
I think it's just the pattern.

Side note:  should use RegisterProtocolsWithContext for new code.